### PR TITLE
E0d/diverse virtualization targets

### PR DESCRIPTION
**Pull request by e0d**
pull restructure for virtualization targets hackathon branch.

### CODE REVIEW

**Comment by e0d** [comment]
@mulby

**Comment by e0d** [comment]
@feanil @mulby @fredsmith

File: playbooks/roles/edxapp/tasks/deploy.yml [hunk 1]
```diff
@@ -399,13 +516,23 @@
...
+  tags:
+    - install
+    - install:configuration
+    - install:code
+  
 - include: tag_ec2.yml tags=deploy
   when: COMMON_TAG_EC2_INSTANCE
+  tags:
+    - remove
+    - aws
```
Comment (mulby) [inline playbooks/roles/edxapp/tasks/deploy.yml:535]: What are these "aws" and "remove" tags?

**Comment by mulby** [comment]
looks reasonable to me
:+1:

**Pull request merged**